app/src/main/java/net/squanchy/tweets/view/TimelineStateHolder.java

```diff
@@ -0,0 +1,55 @@
+package net.squanchy.tweets.view;
+
+import com.twitter.sdk.android.tweetui.TimelineCursor;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class TimelineStateHolder {
+
+    private TimelineCursor nextCursor;
```

Comment:
can we make both of the `TimelineCursor`s optional? so that if someone deals with this code will know they have to check for presence